### DISCUSSION TITLE
同步MD的时候图片存放的路径可以自定义吗？

### DISCUSSION THREAD

**Discussion by janeluyip** [windingwind/zotero-better-notes#780] [Q&A] [answered]
目前笔记同步在同一个文件夹下面的不同.md 文件中。但是笔记文件中的截图全部同步在attachments文件中了，但是不能自定义同步在例如attachment/{{title}}/之类的方式进行同步。现在有这样的功能吗？还是需要更新？

**Comment by windingwind** [comment, marked as answer]
目前没有这样的功能。是否有必要还有待商榷，自定义路径很容易导致附件冲突
